utils/test-utils/README.md

```diff
@@ -0,0 +1,254 @@
+# Testing-kotlin
+
+In this module you can find the different utilities that we have created in order to help us to implement UI tests on our libraries.
+
+These utilities were originally created as a way to reuse code on our different UI tests without the need to copy-paste useful function on each library module.
+Since we think that there is value on these utilities to be shared with the developer community we have created a testing component that can be added as a external dependency to any project that want to implement UI tests targeting dual-screen and foldable devices.
+
+## Add to your project
+
+1. Make sure you have **mavenCentral()** repository in your top level **build.gradle** file:
+
+    ```gradle
+    allprojects {
+        repositories {
+            google()
+            mavenCentral()
+         }
+    }
+    ```
+
+2. Add dependencies to the module-level **build.gradle** file (current version may be different from what's shown here):
+
+    ```gradle
+    implementation "com.microsoft.device.dualscreen.testing:testing-kotlin:1.0.0-alpha1"
+    ```
+
+## API reference
+
+The library is divided into a few classes that cover just one specific purpose:
+
+### CurrentActivityDelegate
+
+[Activity](https://developer.android.com/reference/android/app/Activity) delgate used to setup and support a test scenario.
+It provides useful functionality used whenever you want to ensure that the Activity is running before use any view action or assertion on its UI components.
+
+API functions:
+
+`fun <T : Activity> setup(activityScenarioRule: ActivityScenarioRule<T>)`
+
+Setup the delegate using a given ActivityScenarioRule/
```

Comment:
There is a "/" at the end. Is it intentional?

Reply:
😱 nice catch!

Reply:
Fixed